### ISSUE TITLE
Delete content  not working for Sandboxes with custom path when it's parent folder is a disk drive.

### ISSUE THREAD

**Issue by henrikx** [sandboxie-plus/Sandboxie#139]
I have created a Sandbox with a custom path:
```ini
[test2]
Enabled=y
ConfigLevel=7
AutoRecover=n
BlockNetworkFiles=n
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00FFFF,ttl
FileRootPath=F:\test2
```
however, when I try to delete it's contents in Sandman I get the message "Could not move the Sandbox folder out of the way Can't open destination folder". Meanwhile SbieCtrl can delete the same sandbox just fine. It also marks the sandbox status as "Net Share" which is incorrect as F:\ is a local drive on my computer.

**Comment by Coverlin** [comment]
> I have created a Sandbox with a custom path:
> 
> ```ini
> [test2]
> Enabled=y
> ConfigLevel=7
> AutoRecover=n
> BlockNetworkFiles=n
> RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
> RecoverFolder=%Personal%
> RecoverFolder=%Desktop%
> BorderColor=#00FFFF,ttl
> FileRootPath=F:\test2
> ```
> 
> however, when I try to delete it's contents in Sandman I get the message "Could not move the Sandbox folder out of the way Can't open destination folder". Meanwhile SbieCtrl can delete the same sandbox just fine. It also marks the sandbox status as "Net Share" which is incorrect as F:\ is a local drive on my computer.
 
not what that means but to switch "Net Share" status off  : 
In sandman : right click the box -> sandbox presets and uncheck "allow network shares"
or  right click the box -> sandbox options -> other restrictions  and check the "block network files and folders......."
In sbiectrl : right click the box -> sandbox settings -> restrictions -> network files and check the "block network files and folders......."
Editing the config : add/change BlockNetworkFiles=y to the sandbox

custom fileroothpath for a sandbox other than the global container, sandman deletes the box for me

**Comment by henrikx** [comment]
Ah, disregard that then.

Still can not get it to remove the box though. 
![image](https://user-images.githubusercontent.com/10342989/93515404-d5d90880-f928-11ea-863c-2d20cdf9dbac.png)

**Comment by DavidXanatos** [comment]
I think the issue is that the parent of the box is a drive directly, i will test that scenario and fix it.

**Comment by henrikx** [comment]
@DavidXanatos Can confirm. Everything works when sandbox is in a subfolder.

**Comment by DavidXanatos** [comment]
the issue will be fixed in the next build 0.4.3
